tests/processors/tests.py

```diff
@@ -201,3 +201,27 @@ def test_remove_stacktrace_locals():
     stack = result['exception']['stacktrace']
     for frame in stack:
         assert 'vars' not in frame
+
+
+def test_mark_in_app_frames():
+    data = {
+        'exception': {
+            'stacktrace': [
+                {'module': 'foo'},
+                {'module': 'foo.bar'},
+                {'module': 'foo.bar.baz'},
+                {'module': 'foobar'},
+                {'module': 'foo.bar.bazzinga'},
+            ]
+        }
+    }
+
+    client = Client(include_paths=['foo'], exclude_paths=['foo.bar.baz'])
```

Comment:
re. documenting `exclude_paths` preference:
i mean that if you do the other way around `Client(include_paths=['foo.bar.baz'], exclude_paths=['foo'])),
then `foo.bar.baz` will be excluded (i think?), which might be not so intuitive.